recipes/rilseq/meta.yaml

```
@@ -1,33 +1,35 @@
 {% set name = "rilseq" %}
-{% set version = "0.75" %}
+{% set version = "0.78" %}
 
 package:
   name: "{{ name|lower }}"
   version: "{{ version }}"
 
 source:
   url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/RILseq-{{ version }}.tar.gz"
-  sha256: 969a009f06c4cfcfeb59c5c9f0680e2196b77a55b663fb459f97c0e5cdd3e02a
+  sha256: 7386116c01b399e1c76f6affd8a75d151ab8e10bee70d2852baabffa35b3e8b1
 
 build:
-  number: 0
+  number: 1
   noarch: python
   script: "{{ PYTHON }} -m pip install . --ignore-installed --no-deps -vv"
 
 requirements:
   host:
     - pip
-    - python
+    - python >=3
   run:
     - biopython
     - numpy
     - pysam >=0.14.1
-    - python
+    - python >=3
     - scipy
     - bwa >=0.7.12
-    - samtools >=1.2
+    - samtools >=1.9
 
 test:
+  commands:
+    - samtools 2>&1 | grep "Usage"
```

```suggestion
    - samtools --help
```

samtools recipe use this, so I suppose it may be safer in the long run.